### PR TITLE
docs: simplify setup instructions

### PR DESCRIPTION
**Pull request by jyeshe**
Update to simplify setup instructions according to different envs and to remove remaining references to `make`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,82 +43,54 @@ The middleware is a caching and reporting layer which sits in front of the nodes
...
-
-### A) Sync from genesis block
+Before running it it's recommeded to use a Node database snapshot for faster syncing.
 
-1. Simply run the middleware with `docker-compose up`
+### Database Snapshot
 
-Next time you run the same command it will continue from the last height synced.
+1. Download the one of the full backups from https://downloads.aeternity.io
+2. Create a 'data' directory under the root repo dir and extract the backup to it.
```
Comment (sborrazas) [inline README.md:58]: Perhaps we should mention that it's the `mnesia` directory that should be extracted into the local `data/mnesia` directory

Comment (jyeshe) [reply README.md]: The backup contain the `mnesia` directory. Added a mention as a result.

File: README.md [hunk 2]
```diff
@@ -44,82 +43,54 @@ The middleware is a caching and reporting layer which sits in front of the nodes
...
 Firstly, clone the middleware repo:
 
 `git clone https://github.com/aeternity/ae_mdw && cd ae_mdw`
 
-For syncing on testnet, change ./docker/aeternity.yaml network_id to `ae_uat`.
-
-Following there are two options depending on which block height you want to continue syncing:
-
-### A) Sync from genesis block
+Before running it it's recommeded to use a Node database snapshot for faster syncing.
```
[the remark this answers is not in the document]
Comment (jyeshe) [reply README.md]: done

**Review by sborrazas — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -44,82 +43,54 @@ The middleware is a caching and reporting layer which sits in front of the nodes
...
-Following there are two options depending on which block height you want to continue syncing:
-
-### A) Sync from genesis block
+Before running it, it's recommended to use a Node database snapshot for faster syncing.
 
-1. Simply run the middleware with `docker-compose up`
+### Database Snapshot
 
-Next time you run the same command it will continue from the last height synced.
+1. Download one of the full backups from https://downloads.aeternity.io
```
Comment (thepiwo) [inline README.md:57]: does this work, I assume they are 6.7.0 based and we are still running 6.5.2?

Comment (jyeshe) [reply README.md:57]: It works for first and last testnet ones.

Comment (thepiwo) [reply README.md:57]: cool, good to know

**Review by thepiwo — APPROVED** [review]

**Pull request merged**